### PR TITLE
golden plunger

### PR DESCRIPTION
**Pull request by Ghagliiarghii**
## About the PR

Added a cool new golden version of the plunger for janitor as a loadout reward for 10 hours playtime in janitor.
## Why / Balance

Loadout items are cool, I like rewards for playtime also. For balance changes, this one glows slightly (less than a PDA light) and you can throw it and it will stick to people, meaning you can 'embed' a light on them to see them in the dark, I don't know of another item that does this.
It also can be recycled for gold, but I wouldn't be worried about this since when testing I got like 1 gold bar after recycling about 15 of them and they're only currently obtainable through the loadouts system.

## Media

image
image

https://github.com/space-wizards/space-station-14/assets/68826635/84a197ed-3014-47f1-9a8f-469f096115df

**Changelog**

- add: Added a golden plunger for experienced janitors! Thank you for your hard work keeping the station clean.

### CODE REVIEW

**Comment by lzk228** [comment]
we need more than 10 hours
it should feel like a reward, not to be in every janitor's backpack

**Comment by Ghagliiarghii** [comment]
> we need more than 10 hours it should feel like a reward, not to be in every janitor's backpack

There's some back and forth on a previous PR I was working on - https://github.com/space-wizards/space-station-14/pull/27559 about this. I want things to feel rewarding but not be completely restrictively time gated and there are some maintainers who don't want the high time gates on things even that should feel rewarding.

I agree in principle but don't know where the proper balance point it.

**Comment by IamVelcroboy** [comment]
God tier PR

File: Resources/Prototypes/Loadouts/Jobs/Civilian/janitor.yml [hunk 1]
```diff
@@ -1,3 +1,12 @@
+- type: loadoutEffectGroup
+  id: SeniorJanitorial
+  effects:
+  - !type:JobRequirementLoadoutEffect
+    requirement:
+      !type:RoleTimeRequirement
+      role: JobJanitor
+      time: 36000 #10 hrs
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Loadouts/Jobs/Civilian/janitor.yml:8]:
```suggestion
      time: 180000 #50 hrs
```
this is kinda a silly meme for janitor so i think a high timer is good. gotta distinguish the scrubs somehow.

Comment (Ghagliiarghii) [reply Resources/Prototypes/Loadouts/Jobs/Civilian/janitor.yml:8]: Very good, just wanted a maintainer to give me a good amount of time. How about 52 hours tho, like 52 weeks = 1 work year? I'm going to do 52 hours.

**Review by Ghagliiarghii — COMMENTED** [review]
(I keep using the comment tool on the .dev instead of the commit button)

**Comment by Ghagliiarghii** [comment]
oh changelog was broken rip credit

**Pull request merged**